utils/wasi-nn/download-tflite-fixtures.sh

```diff
@@ -0,0 +1,15 @@
+TODIR=$1
+
+FIXTURE_EX=https://raw.githubusercontent.com/second-state/wasm-learning/master/rust/birds_v1
```

Comment:
Change to use the `WasmEdge-WASINN-examples` repo after [this PR](https://github.com/second-state/WasmEdge-WASINN-examples/pull/12) being merged.